anime-list-master.xml

```diff
@@ -43018,6 +43018,9 @@
   </anime>
   <anime anidbid="15660" tvdbid="398106" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Yakunara Mug Cup mo</name>
+    <mapping-list>
+      <mapping anidbseason="0" tvdbseason="0" start="" end="" offset="-1"></mapping>
```

Comment:
start and end are not needed

Reply:
ok i removed it, i wasn't sure if this is needed even when empty